tools/SeekRPC/SeekRPCClient.py

```diff
@@ -2,8 +2,8 @@
 from thrift.transport import TTransport, TSocket
 from thrift.protocol.TBinaryProtocol import TBinaryProtocol
 
-from pyseek import SeekRPC
-from pyseek.ttypes import SeekQuery, QueryParams, QueryResult
+from pyseek import SeekRPC, constants
```

Comment:
I had problems with this import at first before I added the link: `pyseek -> gen-py/seek_rpc`. Maybe add a comment here to add this symlink (here and at README)